packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1763,6 +1792,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
      * @param options - options controlling how the summary is generated or submitted
      */
     public async submitSummary(options: ISubmitSummaryOptions): Promise<SubmitSummaryResult> {
+        if (this.createContainerMetadata.summaryCount !== undefined) {
+            this.createContainerMetadata.summaryCount++;
+        } else {
+            this.createContainerMetadata.summaryCount = 0;
```

Comment:
I'd rather move this initialization to constructor (on !existing path - if it's not there, just put zero) and thus make summaryCount non-optional